### DISCUSSION TITLE
families with the largest difference

### DISCUSSION THREAD

**Discussion by iaunicorn** [hahnlab/CAFE5#122] [Q&A] [answered]
Hi,

When I ran cafe5, I met this error: "You may want to try removing the top few families with the largest difference between the max and min counts and then re-run the analysis. Failed to initialize any reasonable values".
Then I removed the top few families, but it still reported that other families with the same problem.
Could you give me some clues? What should I do? Keep removing until it does give any error?

Thanks.

**Comment by hahnlab-user** [comment, marked as answer]
Yes, I would recommend removing more families. Alternatively, it may be that your tree is too deep. In this case, you could keep all your families but divide your analyses into multiple sets of more-closely related species.

**Comment by iaunicorn** [comment]
Thanks a lot for the reply and I will try. 
One more question: if I would like to know the expansion and contraction of a small group of genes in one certain Orthogroup, what should I do?

**Comment by benfulton** [comment]
The program [CafePlotter](https://github.com/moshi4/CafePlotter) will show you expansions and contractions in your tree.
